src/services/iiif-parser.js

```diff
@@ -24,7 +33,7 @@ export function filterVisibleRangeItem(item) {
   const behavior = manifesto
     .create(getReduxManifest())
     .getRangeById(item.id)
-    .getBehavior();
+    .getViewingHint();
```

Comment:
Why did you switch to `viewingHint`?  I think this should be `behavior` based upon how it is used below and the spec: https://iiif.io/api/presentation/3.0/#behavior
If our manifests have it in a different place then we should probably fix the manifests and not try to handle it here.

Reply:
The iiif-manifest I got has `viewingHint` instead of `behavior` in the structure.
In the previous manifest used;
```
structures: [
    {
      type: 'Range',
      id:
        'https://pawpaw.dlib.indiana.edu/media_objects/2j62s484w/manifest/range/rb001ef98-954a-47c2-8690-28ab63ae867d',
      label: null,
      behavior: 'top',
      items: [
        {
```

The latest one I got;
```
structures: [
    {
      id: 'https://dlib.indiana.edu/iiif_av/mahler-symphony-3/range/0',
      type: 'Range',
      viewingHint: 'top',
      label: {
        en: ['Symphony no. 3 - Mahler, Gustav']
      },
      items: [
        {
```
That is why I got the idea to switch it from `getBehavior()` to `getViewingHint()`.

Reply:
Ah, ok.  I don't think these are really the same thing.  Here is viewingHint in the Presentation 2.1 spec: https://iiif.io/api/presentation/2.1/#viewinghint .  But it looks like viewingHint was folded into behavior in the 3.0 spec.  Maybe there is a change that should be made in `iiif_manifest`. 🤔 
So... I'd stick with behavior here in the player.

Reply:
Should I change the manifest accordingly?

Reply:
👍 That sounds good to me.